config/standalone.example.toml

```diff
@@ -96,7 +96,7 @@ sync_write = false
 # Storage type.
 type = "File"
 # Data directory, "/tmp/greptimedb/data" by default.
-data_dir = "/tmp/greptimedb/data/"
+data_home = "/tmp/greptimedb/"
```

Comment:
It seems a little weird that `data_home` configured under this section can be inherited by other section `wal`

Reply:
Agree, but I can't find a better place to put it.

Reply:
Could we put data_home in the `StorageConfig` and make `data_dir` optional in ObjectStoreConfig?
```
pub struct StorageConfig {
    data_home: String,
    #[serde(flatten)]
    pub store: ObjectStoreConfig,
    pub compaction: CompactionConfig,
    pub manifest: RegionManifestConfig,
    pub flush: FlushConfig,
}
```

We can construct the right `data_dir` from the `data_home` later.

Reply:
If we allow the user configure the `data_dir`, then we will need two `ObjectStore` instances: one for internal states such as procedure state, and another for table data.

Reply:
It seems only FileConfig has data_home? I just wonder if the data_home in the ObjectStoreConfig is confusing since we may reuse the ObjectStoreConfig.

Could we just ignore the `data_dir` in this case? We always overwrite it with our `data_home` after we finish parsing the StorageConfig. Or could we [skip](https://serde.rs/field-attrs.html#skip) `data_dir` during serializing?

Reply:
As we discussed, keep using `data_home`.